server/bin/pbench-server-prep-shim-002.py

```diff
@@ -11,97 +11,66 @@
 import os
 import sys
 import glob
-import shutil
-import selinux
-import tempfile
 
+from logging import Logger
 from pathlib import Path
 
 from pbench.common.exceptions import BadConfig
 from pbench.common.logger import get_pbench_logger
-from pbench.common.utils import md5sum
 from pbench.server import PbenchServerConfig
-from pbench.server.report import Report
-from pbench.server.utils import quarantine
-from pbench.server.database.models.tracker import Dataset, States, DatasetError
-from pbench.server.database import init_db
+from pbench.cli.agent.commands.results import MoveResults
+from pbench.agent.results import FileUploadError
 
 
 _NAME_ = "pbench-server-prep-shim-002"
 
 
 class Results:
-    def __init__(
-        self, nstatus="", ntotal=0, ntbs=0, nquarantined=0, ndups=0, nerrs=0,
-    ):
+    """Results: Class to collect status details about tar ball
+
+        Args -
+            nstatus -- appends string status of tarball
+            ntotal -- total number of tarballs processed
+            ntbs -- number of tarballs successfully processed
+            nerrs -- number of tarballs that failed
+    """
+
+    def __init__(self, nerrs=0, nstatus="", ntotal=0, ntbs=0):
+        self.nerrs = nerrs
         self.nstatus = nstatus
         self.ntotal = ntotal
         self.ntbs = ntbs
-        self.nquarantined = nquarantined
-        self.ndups = ndups
-        self.nerrs = nerrs
 
 
-def fetch_config_val(config, logger):
-
-    qdir = config.get("pbench-server", "pbench-quarantine-dir")
-    if not qdir:
-        logger.error("Failed: getconf.py pbench-quarantine-dir pbench-server")
-        return None, None
-
-    qdir = Path(qdir).resolve()
-    if not qdir.is_dir():
-        logger.error("Failed: {} does not exist, or is not a directory", qdir)
-        return None, None
+def fetch_config_val(config: PbenchServerConfig, logger: Logger) -> Path:
 
     # we are explicitly handling version-002 data in this shim
     receive_dir_prefix = config.get("pbench-server", "pbench-receive-dir-prefix")
     if not receive_dir_prefix:
         logger.error("Failed: getconf.py pbench-receive-dir-prefix pbench-server")
```

Comment:
This `getconf.py` reference must be a leftover from the original shell script, and even our shell scripts no longer use `getconf.py`. Maybe just "Failed to get configuration option {} {}"???

Reply:
yeah, it must have been a leftover. Will change this.